<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HodlClaimWallet</class>
 <widget class="QDialog" name="HodlClaimWallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>240</y>
     <width>652</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="coldWalletKeyHashLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="control::MwcLabelSmall" name="coldWalletPublicKeyHashLable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>Cold Wallet root public key hash</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="control::MwcLineEditNormal" name="coldWalletPublicKeyHash">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Cold Wallet root public key hash</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>335</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>75</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>It is possible to claim MWC for another wallet (Cold Wallet).
Funds will be sent to this wallet.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="continueButton">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>335</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>25</y>
     <width>280</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>HODL Claim</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="claimForThisWalletCheck">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>155</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>I am claiming MWC for this wallet</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="coldWalletKeyHashLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>200</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: You will need an access to your Cold wallet during claiming process.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>claimForThisWalletCheck</tabstop>
  <tabstop>coldWalletPublicKeyHash</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>continueButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
